staging/src/k8s.io/apiserver/pkg/storage/etcd3/store_test.go

```diff
@@ -64,7 +67,7 @@ func init() {
 	utilruntime.Must(example.AddToScheme(scheme))
 	utilruntime.Must(examplev1.AddToScheme(scheme))
 
-	capnslog.SetGlobalLogLevel(capnslog.CRITICAL)
+	clientv3.SetLogger(grpclog.NewLoggerV2(ioutil.Discard, ioutil.Discard, os.Stderr))
```

Comment:
Let's use directly: 
  grpclog.SetLoggerV2(grpclog.NewLoggerV2(ioutil.Discard, ioutil.Discard, os.Stderr))

Reply:
done in the later commit (Drop use of deprecated clientv3.SetLogger)